LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_pointer_boundary_events_after_reappending_last_over_target-expected.txt

```diff
@@ -1,5 +1,5 @@
 
-FAIL After re-appending the last over element at pointerover, pointer boundary events should be fired on the original target again assert_equals: expected "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointerover@div#child, pointerenter@div#child, pointermove@div#child" but got "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointermove@div#child"
-FAIL After re-appending the last over element at pointerenter, pointer boundary events should be fired on the original target again assert_equals: expected "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointerover@div#child, pointerenter@div#child, pointermove@div#child" but got "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointermove@div#child"
+FAIL After re-appending the last over element at pointerover, pointer boundary events should be fired on the original target again assert_equals: expected "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointerover@div#child, pointerenter@div#child, pointermove@div#child" but got "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#parent, pointermove@div#child"
+FAIL After re-appending the last over element at pointerenter, pointer boundary events should be fired on the original target again assert_equals: expected "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#child, pointerover@div#child, pointerenter@div#child, pointermove@div#child" but got "pointerover@div#child, pointerenter@body, pointerenter@div#grandparent, pointerenter@div#parent, pointerenter@div#child, pointermove@div#parent, pointermove@div#child"
```

Comment:
These may be easy candidates for a coming progression? We fire an unnecessary `pointermove@div#parent`